prime-angular-frontend/src/app/modules/enrolment/pages/access-agreement/access-agreement.component.html

```diff
@@ -6,10 +6,18 @@
       <app-page-header>Access Agreement</app-page-header>
 
       <ng-container *ngIf="currentPage > 0; else preamble">
-        <!-- <app-moa-access-agreement [currentPage]="currentPage"
-                                      (changed)="onPageChange($event)"></app-moa-access-agreement> -->
-        <app-ru-access-agreement [currentPage]="currentPage"
-                                 (changed)="onPageChange($event)"></app-ru-access-agreement>
+
+        <ng-container *ngIf="enrolment.enrolleeClassification === 'MOA'">
```

Comment:
An enumeration for enrollee classification would be useful now and in the future